source/types/ut_test.tpb

```diff
@@ -14,29 +14,23 @@ create or replace type body ut_test is
 
   member function is_valid(self in ut_test) return boolean is
   begin
-    if call_params.test_procedure is null then
-      return false;
-    end if;
-  
-    if not ut_metadata.resolvable(call_params.owner_name, call_params.object_name, call_params.test_procedure) then
-      return false;
-    end if;
-  
-    if call_params.setup_procedure is not null and
-       not ut_metadata.resolvable(call_params.owner_name, call_params.object_name, call_params.setup_procedure) then
-      return false;
-    end if;
-  
-    if call_params.teardown_procedure is not null and
-       not ut_metadata.resolvable(call_params.owner_name, call_params.object_name, call_params.teardown_procedure) then
-      return false;
-    end if;
-  
-    return true;
+    return case
+      when call_params.test_procedure is null then false
+      when not ut_metadata.resolvable(call_params.owner_name, call_params.object_name, call_params.test_procedure) then false
+      when call_params.setup_procedure is not null and
+       not ut_metadata.resolvable(call_params.owner_name, call_params.object_name, call_params.setup_procedure) then false
+      when call_params.teardown_procedure is not null and
+       not ut_metadata.resolvable(call_params.owner_name, call_params.object_name, call_params.teardown_procedure) then false
+      else true
```

Comment:
Logic regression. To eliminate multiple returns should be rewritten as:
`return call_params.test_procedure is not null 
and ut_metadata.resolvable(call_params.owner_name, call_params.object_name, call_params.test_procedure) 
and (call_params.setup_procedure is null OR ut_metadata.resolvable(call_params.owner_name, call_params.object_name, call_params.setup_procedure))
and (call_params.teardown_procedure is null OR ut_metadata.resolvable(call_params.owner_name, call_params.object_name, call_params.teardown_procedure));`